drivers/media/i2c/imx296.c

```diff
@@ -597,6 +646,10 @@ static int imx296_stream_on(struct imx296 *sensor)
 	usleep_range(2000, 5000);
 	imx296_write(sensor, IMX296_CTRL0A, 0, &ret);
 
+	/* vflip and hflip cannot change during streaming */
+	__v4l2_ctrl_grab(sensor->vflip, 1);
```

Comment:
Nothing is mentioned in the datasheet.  But I would imagine there may be some timing related issues doing it mid-frame.  To be save, I would consider keeping this as-is.

Reply:
I only raised it as I had the same question on [imx290](https://patchwork.linuxtv.org/project/linux-media/patch/20230131192016.3476937-12-dave.stevenson@raspberrypi.com/#144355) where it is explicitly stated in the datasheet that it will create a corrupt frame, however that is a rolling shutter sensor compared to global.